tests/unit/api/bucketPut.js

```diff
@@ -338,6 +351,26 @@ describe('bucketPut API', () => {
         });
     });
 
+    it('should deny put bucket to cold storage', done => {
+        const bucketName = 'cold-bucket-name';
+        const newRestEndpoint = 'location-dmf-v1';
+        const coldLocation = 'location-dmf-v1';
+
+        const req = Object.assign({}, testRequest, {
+            parsedHost: newRestEndpoint,
+            bucketName,
+        });
```

Comment:
Use spread over object assign now that we are on Node 16
```suggestion
        const req = {
            ...testRequest,
            parsedHost: newRestEndpoint,
            bucketName,
        });
```
( I think this is the syntax )

Reply:
Done: https://github.com/scality/cloudserver/pull/4532/files#diff-8650e98577b5216380b7fe846ee48f198b9b0dfd793cac14c405a9aa61f43770R360